<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="afficher_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>451</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget:pane {
border: 1px solid gray;
}
QTabWidget::pane {
  border: 1px solid lightgray;
  top:-1px; 
  background:rgb(0, 170, 255); 
} 

QTabBar::tab {
  background:rgb(255, 0, 0); 
  border: 1px rgb(0, 85, 255); 
  padding: 15px;
} 

QTabBar::tab:selected { 
  background: rgb(245, 245, 245); 
  margin-bottom: -1px; 
}
QTabWidget {  
    background-color:#0f2033;
    border-top: 2px solid #C2C7CB;  
} 
QTabWidget::pane { /* The tab widget frame */  
    border-top: 3px solid #C2C7CB;  
} 
 /* Style the tab using the tab sub-control. Note that 
    it reads QTabBar _not_ QTabWidget */  
QTabBar::tab {  
    background-color:#0f2033;
    border: 1px solid #4973b4;  
    min-width: 100px; 
    min-height:22px;
    padding: 0px  2px; 
    color: rgb(255, 255, 255);
    margin-left: 1px;
    font: 75 12pt &quot;NSimSun&quot;;
}  
  
QTabBar::tab:selected, QTabBar::tab:hover {  
    background-color:rgb(0, 0, 255);
    margin-top: 0px;
    
}  
  

  
QTabBar::tab:!selected {  
    margin-top: 5px; /* make non-selected tabs look smaller */  
}  

</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_20">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLabel" name="label_60">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>id d'audit </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_61">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Date d'audit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_62">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Dur√©e </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_63">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Remarque</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="duree">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="remarque">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>124</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 120px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="resultat">
      <property name="geometry">
       <rect>
        <x>222</x>
        <y>260</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_64">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Resultat</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_21">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_65">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>           id d'audit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>124</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 120px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxAuditSupp">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_22">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLabel" name="label_66">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>id d'audit </string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>124</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 120px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_dest_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_67">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Date d'audit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_num_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_68">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Remarque</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_69">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Dur√©e </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_agence_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_70">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
      </property>
      <property name="text">
       <string>Resultat</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateM">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QPushButton" name="afficher">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>124</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 120px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableAff">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>11</y>
        <width>441</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>184</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 180px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
    </property>
    <property name="text">
     <string>Generation PDF</string>
    </property>
   </widget>
   <widget class="QTableView" name="pdf">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QTableView {
     selection-background-color: qlineargradient(rgb(0, 170, 255));
font: 75 12pt &quot;NSimSun&quot;;
 }
 QTableView QTableCornerButton::section {
     background:  rgb(0, 0, 255);
     border: 2px outset  rgb(0, 0, 255);
 }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_71">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
    </property>
    <property name="text">
     <string>Trier par :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tri">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>340</y>
      <width>284</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 280px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
    </property>
    <property name="text">
     <string>date d'audit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stat">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>284</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 280px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
    </property>
    <property name="text">
     <string>statistique</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recherche">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>124</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 75 ;
padding-left: 1px;
padding-right: 1px;
min-width: 120px;
max-width: 50px;
min-height: 15px;
max-height: 15px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
